scripts/sign.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config()
+
+exports.default = async function(conf) {
+  // do not include passwords or other sensitive data in the file
+  // rather create environment variables with sensitive data
+  const tokenPassword = process.env.CSC_KEY_PASSWORD
+
+  if (conf.hash === 'sha1') return
+  if (require('os').platform() !== 'win32') {
+    console.log(`SKIPPING WINDOWS SIGNING ${conf.path}`)
+    return
+  }
+
+  console.log(`Signing Windows Binary ${conf.path}`)
```

Comment:
No important but this is probably cleaner:

```suggestion
  console.log('Signing Windows Binary', conf.path)
```